### PR TITLE
Solution for `React Article`

### PR DESCRIPTION
**Pull request by jstmpelowycz**
[DEMO LINK](https://jstmpelowycz.github.io/react_article/)

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title,
+  text,
+  date,
+}) => (
```
Comment (Andr1yK) [inline src/components/Article.jsx:8]:
```suggestion
export const Article = ({
  title, text, date,
}) => (
```

File: src/App.jsx [hunk 2]
```diff
@@ -8,8 +10,8 @@ const article = {
   date: 'August 30, 2019',
 };
 
-const App = () => (
-  <h1>React Article</h1>
+export const App = () => (
+  <>
+    {Article(article)}
```
Comment (Andr1yK) [inline src/App.jsx:15]:
```suggestion
   <Article {...article} />
```

**Review by Andr1yK — CHANGES_REQUESTED** [review]
Good start, but fix the App.jsx file 😀

File: src/App.jsx [hunk 3]
```diff
@@ -8,8 +10,8 @@ const article = {
   date: 'August 30, 2019',
 };
 
-const App = () => (
-  <h1>React Article</h1>
+export const App = () => (
+  <>
+   <Article {...article} />
```
Comment (Ihor-Karpyn) [inline src/App.jsx:15]:
```suggestion
   <Article article={article} />
```

File: src/components/Article.jsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = ({
+  title, text, date,
+}) => (
```
Comment (Ihor-Karpyn) [inline src/components/Article.jsx:6]:
```suggestion
export const Article = ({ title, text, date }) => (
```

**Review by Ihor-Karpyn — CHANGES_REQUESTED** [review]

**Review by Ihor-Karpyn — APPROVED** [review]

**Pull request closed without merging**